doc/source/examining/loading_data.rst

```diff
@@ -2111,6 +2111,33 @@ For example, to add support for a longint field named
 ``my_custom_field``, one would add ``('my_custom_field', 'l')`` to ``assoc``.
 
 
+Particle ages and formation times
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For cosmological simulations that include star particles, RAMSES stores particle
+formation times as conformal times. To access these raw data in conformal units
+use the ``"conformal_formation_time"`` field. This will return the formation
+times of particles in the simulation in conformal units as a dimensionless
+array. To access the formation time in physical units, use the
+``"particle_formation_time"`` field. Finally, to access the ages of star
+particles in your simulation, use the ``"star_age"`` field. Note that this field
+is defined for all particle types but will only make sense for star particles.
+
+To avoid confusion, it may be convenient to define a particle filter to
```

Comment:
Maybe it'd be great to add a table listing all the `age` fields, something like

| Simulation type  | Field name                           | Description                                   |
|------------------|-------------------------------------|-------------------------------------------|
| cosmological | `conformal_formation_time` | Formation time in conformal unit (dimensionless) |
| any               | `particle_formation_time`      | Formation time, negative or null (???).                  |
| any               | `star_age`                             | Age of the star, positive. Only relevant for stars.    |
| any               | `particle_age`                        | *Depreciated*                                                         |


It is also unclear what the origin of time is some cases, e.g. for the `particle_formation_time`.